### PR TITLE
IE10

### PR DESCRIPTION
**Pull request by matthew-andrews**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -2,7 +2,7 @@ API_KEY := $(shell cat ~/.ftapi 2>/dev/null)
 API2_KEY := $(shell cat ~/.ftapi_v2 2>/dev/null)
 GIT_HASH := $(shell git rev-parse --short HEAD)
 TEST_HOST := "ft-next-dobi-branch-${GIT_HASH}"
-TEST_URL := "http://ft-next-dobi-branch-${GIT_HASH}.herokuapp.com/"
+TEST_URL := "ft-next-dobi-branch-${GIT_HASH}.herokuapp.com"
```
Comment (matthew-andrews) [inline Makefile]: i'm also going to…… at some point…… tidy up these env vars…… the host is not a host and this is no longer a URL and it was always not the TEST URL…………

**Comment by i-like-robots** [comment]
Definitely needs more commits! If CI passes both times then let's do it.

**Comment by i-like-robots** [comment]
Keen.

**Comment by matthew-andrews** [comment]
soz

**Comment by matthew-andrews** [comment]
i recreated something suspiciously like the bug we saw

**Comment by matthew-andrews** [comment]
and fixed it

**Comment by matthew-andrews** [comment]
if it happens again just add the `--env` option to the `nightwatch` test and i'll look into it again :innocent:

**Pull request merged**